# Well, since asm/page.h has been removed from make headers_install,
# we replace it with sys/user.h
# Copyright 2010, mario <mario@slackverse.org>

--- mbootpack-0.6a/buildimage.c.ORIG	2008-12-09 14:10:59.000000000 +0100
+++ mbootpack-0.6a/buildimage.c	2010-06-18 18:15:11.130697044 +0200
@@ -38,7 +38,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
-#include <asm/page.h>
+#include <sys/user.h>
 
 #include "mbootpack.h"
 #include "mb_header.h"
--- mbootpack-0.6a/mbootpack.c.ORIG	2008-12-09 14:10:59.000000000 +0100
+++ mbootpack-0.6a/mbootpack.c	2010-06-18 18:15:21.709696839 +0200
@@ -43,7 +43,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
-#include <asm/page.h>
+#include <sys/user.h>
 
 /* From GNU GRUB */
 #include "mb_header.h"
--- mbootpack-0.6a/setup.S.ORIG	2008-12-09 14:10:59.000000000 +0100
+++ mbootpack-0.6a/setup.S	2010-06-18 18:15:33.899696996 +0200
@@ -82,7 +82,7 @@
 #include <linux/compile.h>
 #include <asm/boot.h>
 #include <asm/e820.h>
-#include <asm/page.h>
+#include <sys/user.h>
 */
 
 /* Definitions that should have come from these includes */
